### ISSUE TITLE
Is `role="slider"`, `aria-valuemax` and `aria-valuemin` needed?

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-slider#191]
### Subject of the issue
[Mozilla ARIA slider role best practice](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role#best_practices) states _HTML's `<input type="range">` implicitly has the `role` of `slider`. Do not use `aria-valuemax` or `aria-valuemin` attributes on `<input type="range">` elements; use `min` and `max` instead. Otherwise, any global `aria-*` attributes and any other `aria-*` attributes applicable to the slider role._

Slider already uses `min` and `max` properties but maybe we should remove `role="slider"`, `aria-valuemax` and `aria-valuemin` as suggested?

**Comment by kirsty-hames** [comment]
I can confirm removing `role="slider"`, `aria-valuemax`, `aria-valuemin` and `aria-valuenow` has no impact and works as expected for voiceOver. Testing on both Mac and iPhone Safari. I'd need someone to confirm other screen readers/environments.

**Comment by oliverfoster** [comment]
I think I only added them because we were having issues with rangeslider and I wanted to make it explicit as to how the slider becomes accessible. 

https://github.com/adaptlearning/adapt-contrib-slider/pull/133/commits/578ad47afc47315395a5632cba8abed9093ce5ed

They are implied properties of the `<input type="range">` element so it should be fine to remove.
